pkg/policy/action.go

```diff
@@ -75,9 +75,6 @@ const (
 	// ListMultipartUploadPartsAction - ListParts Rest API action.
 	ListMultipartUploadPartsAction = "s3:ListMultipartUploadParts"
 
-	// ListObjectsAction - ListObjects Rest API action exactly same behavior as ListBucketAction.
-	ListObjectsAction = "s3:ListObjects"
```

Comment:
keep this action commented to denote its not supported in AWS S3.  So that it will be helpful for code reader than missing action.

Reply:
> I am sure the action is not supported in AWS S3 and should be supported in some other service(s)

`s3:ListObjects` i.e list-objects is not relevant to anything other than S3. It is neither documented nor supported by AWS-S3. IMO it is better not to keep this. If you are particular then I can add it so that we can move forward and merge the PR.

Reply:
Mentioned link https://docs.aws.amazon.com/IAM/latest/UserGuide/list_s3.html is redirecting to https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazons3.html now.  In a new page, `s3:ListObjects` is removed.

You would need to update the link?!